src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -104,18 +109,33 @@
     <None Remove="Views\Menu\TrustedPathView.xaml" />
     <None Remove="Views\GuidedTour\HtmlPages\Resources\ConnectTheNode.gif" />
   </ItemGroup>
-  <ItemGroup>
+    <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0-windows' ">
+        <PackageReference Include="System.Configuration.ConfigurationManager" Version="5.0.0" />
+        <PackageReference Include="AvalonEdit" Version="6.3.0.90" CopyXML="true" />
+        <PackageReference Include="HelixToolkit" Version="2.24.0" />
+        <PackageReference Include="HelixToolkit.Wpf" Version="2.24.0" />
+        <PackageReference Include="HelixToolkit.Wpf.SharpDX" Version="2.24.0" />
+        <Reference Include="PresentationCore" /> 
+    </ItemGroup>
+    <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">
+        <PackageReference Include="AvalonEdit" Version="4.3.1.9430" CopyXML="true" />
+        <Reference Include="System.Configuration" />
+        <PackageReference Include="HelixToolkit" Version="2.17.0" />
```

Comment:
why use an old version of helix for net48, why not unify them?

Reply:
It can be unified, but in this moment, if we unify, we are gonna affect other projects that are not migrated yet like DynamoCoreWpfTests or Watch3DNodeModelsWpf. This happened when I try to unify.
<img width="651" alt="Screenshot 2023-06-21 093740" src="https://github.com/DynamoDS/Dynamo/assets/134449953/b9d53d3a-df82-4eec-8fd3-a506662ca5a9">

Reply:
ah I see - ok.